### PR TITLE
add item index and scrollable area on mobile

### PR DESCRIPTION
**Pull request by salieflewis**

### CODE REVIEW

**Comment by salieflewis** [comment]
Resolves #452

File: apps/site/components/server/ItemSidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+interface ItemSidebarProps {
+  itemContext: Adds
+  itemMetadata: {
+    name: string
+    description: string
+    image: string
+    animationUri: string
+    contentType: string
+    muxAssetId?: string
```
Comment (0xTranqui) [inline apps/site/components/server/ItemSidebar.tsx:18]: dont need. muxasset id + mux playback props in here? unless it makes it easier to pass data thru, dont think these get used anywhere

File: apps/site/components/server/ItemSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  return (
+    <div className="overflow-y-auto md:overflow-hidden">
+      {channel?.adds?.items?.map((add: Adds, index: number) =>
+        add.removed ? null : (
+          <Link
+            href={`/channel/${add.channelId}/${totalItems - index}`}
+            className={`${styles.tableRow}`}
+          >
+            <Flex className="gap-4 py-2 items-center hover:cursor-pointer">
+              <ThumbnailNameCreator item={add.item} metadata={metadata} />
```
Comment (0xTranqui) [inline apps/site/components/server/ItemSidebar.tsx:117]: love that we can resuse thumbnail name creator comp right here

**Review by 0xTranqui — APPROVED** [review]
couple small things but approved overall. before u merge, would love to have a conversation bout the following:

feel VERY strongly that the`index `view should be the default, rather than the `information` view. i think the only thing missing for that to happen imo is some way to surface the channel ur in on that `index` view, since rn theres no way to nav back to the channel when looking at that view.

File: apps/site/components/server/ItemSidebar.tsx (hunk 2 above)
Comment (salieflewis) [reply apps/site/components/server/ItemSidebar.tsx:117]: Yeah that was super nice

**Comment by ioeylim** [comment]
also noting that the distance between each row is larger than on the list view in the channel page

**Pull request merged**